virtualenv.py

```diff
@@ -54,7 +54,8 @@
 
 is_jython = sys.platform.startswith('java')
 is_pypy = hasattr(sys, 'pypy_version_info')
-is_win = (sys.platform == 'win32')
+is_win = sys.platform == 'win32' or os.name == 'nt' or getattr(os, '_name', None) == 'nt'
+is_cpython = not is_jython and not is_pypy
```

Comment:
I'd be much happier with an affirmative definition of cpython, rather than "well it hasn't identified itself as alternatives... so probably cpython?" complement approach. If not possible, then you could rearrange the branching logic to be easier on the eyes through negating the conditions

Reply:
I haven't found a way to determine that cpython is the one being run on. I will try and research this, as it would definitely be cleaner.